saleor/product/tests/test_product_availability.py

```diff
@@ -22,12 +22,14 @@ def test_availability(stock, monkeypatch, settings, channel_USD):
     monkeypatch.setattr(
         PluginsManager, "apply_taxes_to_product", Mock(return_value=taxed_price)
     )
+    manager = get_plugins_manager()
     availability = get_product_availability(
         product=product,
         product_channel_listing=product_channel_listing,
         variants=product.variants.all(),
         variants_channel_listing=variants_channel_listing,
         channel=channel_USD,
+        plugins=manager,
```

Comment:
In `get_product_availability` we could also rename `plugins` to `manager` for consistency.